datasets/simulated.py

```diff
@@ -0,0 +1,25 @@
+from benchopt import safe_import_context
+from benchopt import BaseDataset
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+
+
+class Dataset(BaseDataset):
+
+    name = "square"
+    install_cmd = "conda"
+    requirements = ["pip:numpy"]
+
+    # List of parameters to generate the datasets. The benchmark will consider
+    # the cross product for each key in the dictionary.
+    parameters = {
+        "dimension": [2, 10],
+    }
+
+    def __init__(self, dimension=2):
+        self.function = lambda x: np.linalg.norm(x, 2)
```

Comment:
Can you just check that it should not be 
```suggestion
        self.function = lambda x: np.linalg.norm(x, 2) ** 2
```